UI/window-basic-main-dropfiles.cpp

```diff
@@ -5,6 +5,7 @@
 #include <QFileInfo>
 #include <QMimeData>
 #include <QUrlQuery>
+#include <QSettings>
```

Comment:
If the rest of the code is behind a `#ifdef _WIN32` then this should be too.

Reply:
Fixed.